$OpenBSD: patch-test_js,v 1.1 2012/06/28 17:27:51 jeremy Exp $

Don't change NODE_PATH in regression tests, since build is done
in separate directory.

--- test.js.orig	Tue May 22 13:28:22 2012
+++ test.js	Tue May 22 13:28:34 2012
@@ -4,7 +4,7 @@ var spawn = require('child_process').spawn;
 var path = require('path');
 
 function runTest(test, cb) {
-	var proc = spawn(process.execPath, [path.join('test', test)], {env: {NODE_PATH: __dirname}});
+	var proc = spawn(process.execPath, [path.join('test', test)]);
 	proc.stdout.setEncoding('utf8');
 	proc.stderr.setEncoding('utf8');
 
